src/west/app/project.py

```diff
@@ -867,6 +867,14 @@ def fetch_missing_imports(self, args):
         self.manifest = Manifest.from_file(topdir=self.topdir,
                                            importer=self.update_importer)
 
+    def update_submodules(self, submodules, call_location):
+        # Updates given list of submodules, using calling git submodule update --init
+        # from the location passed through the call_location parameter value
+        if submodules:
+            # iterate through the submodules list and update each one of them
+            for submodule in submodules:
+                project.git('-C ' + call_location + ' submodule update --init ' + submodule['path'])
```

Comment:
Consider: 
```suggestion
                project.git(['-C ', call_location, 'submodule update --init ', submodule['path']])
```

Reply:
It is almost good, because 'submodule update --init' must be split into separate commands, otherwise it will not work. Thanks for suggestion :+1: